angular/core/components/thread_page/context_panel/context_panel.scss

```diff
@@ -28,25 +28,22 @@
   margin-bottom: 0;
 }
 
-.context-panel__attachments,
+.context-panel__attachments {
+  margin: 16px 0;
+}
+
 .context-panel__attachment {
   margin-top: 16px;
 }
 
+.context-panel__actions {
+  display: flex;
```

Comment:
Have a look at this on safari; I know we've had issues with just `display: flex` in the past. I think `include @displayFlex;` will take care of it?